### ISSUE TITLE
Investigate and implement any changes required for Poppy's transfer of authority information from mod-inventory-storage to mod-entities-links

### ISSUE THREAD

**Issue by hudajkhan** [sul-dlss/folio-graphql#169]
As described on this page, "Authorities, authority note types, authority source files, and the authority reindex API have been relocated from mod-inventory-storage to mod-entities-links."
https://folio-org.atlassian.net/wiki/spaces/REL/pages/16319169/Changes+and+required+actions

We should review what changes to our FOLIO GraphQL schema files we should make. 

Stemming from https://github.com/orgs/sul-dlss/projects/60/views/1?pane=issue&itemId=50152619

**Comment by hudajkhan** [comment]
- In SUL FOLIO GraphQL, we still include authority information under mod-inventory-storage in json schemas: https://github.com/sul-dlss/folio-graphql/tree/main/json-schemas/mod-inventory-storage/ramls/authorities.
- SUL FOLIO GraphQL autogenerates GraphQL schema from src files that are retrieved by cloning FOLIO Git repositories that contain JSON schema files: https://github.com/sul-dlss/folio-graphql/?tab=readme-ov-file#updating-the-folio-schemas. The cloning of the files and the copying into JSON schemas is a manual process and not automated.
- We also do not currently have mod-entities-links represented under json-schemas. In FOLIO: https://s3.amazonaws.com/foliodocs/api/mod-entities-links/s/mod-entities-links.html.
- The Authority class is at the root level of the GraphQL schema and is referenced for example under InstanceContributorItem.

**Comment by hudajkhan** [comment]
Findings: 

- Although we got authority information to work with the new endpoint, we realized we would have to do YAML to JSON conversion for mod-entities-links which does not include RAMLS.  @shelleydoljack confirmed with us that FOLIO devs are moving away from RAMLS to Swagger. 
- We also didn't find the authority class being requested in any downstream applications (e.g SearchWorks Traject Indexer, Requests, My Library). 
- After reviewing this with @marlo-longley and @cbeer , we have decided to pull out the authority information currently (since the authority info has moved from mod-inventory-storage,we need to work on getting the mod-entities-links information, and the authority class level info is not used in any downstream applications). 

Closed by #172
